code/game/objects/items/storage/boxes/clothes_boxes.dm

```diff
@@ -73,6 +83,12 @@
 	new /obj/item/tank/internals/oxygen(src)
 	new /obj/item/gps(src)
 
+/obj/item/storage/box/hero/PopulateContents()
+	new /obj/item/clothing/head/wizard/tape/fake(src)
+	new /obj/item/clothing/suit/wizrobe/tape/fake(src)
+	new /obj/item/staff/tape(src)
+	new /obj/item/stack/sticky_tape(src)
```

Comment:
what are you doing here

Reply:
Being stupid and removing the wrong thing, whoops